### DISCUSSION TITLE
Hot CUe Colours in XML

### DISCUSSION THREAD

**Discussion by mrdavet247** [dylanljones/pyrekordbox#119] [General]
I noticed that the Hot Cue colours appear in the XML file. Do you plan to support accessing these part of xml.get_track function?

**Comment by dylanljones** [comment]
Hey! What Rekordbox version are you using? Must be a new feature. Sadly the official XML docs don't mention this yet. But I can look into this and add it to the XML handler!

**Reply by mrdavet247** [reply]
Hi

I'm using 6.8.1... Yeah for sure, I couldn't find it documented also. I've pasted a sample file with shows the RGB elements in the XML. I don't colour Memeory Cues, so not sure if teh is a Hot Cue feature only. Appriciate you adding this, thank you!

[Rekordbox.txt](https://github.com/dylanljones/pyrekordbox/files/14281101/Rekordbox.txt)
